src/main/resources/templates/welcome.html

```diff
@@ -100,6 +99,23 @@
                     </ul>
                 </div>
             </div>
+            <div class="col-12 col-lg-7">
+                <div class="border border-dark thank-you">
+                    Do you wonder: what can be a secret? It is often a confidential piece of information which is required to unlock certain functionalities or information.
+                    A secret exists in many shapes or forms, for instance:
```

Comment:
```suggestion
                    Wondering what a secret is? A secret is often a confidential piece of information that is required to unlock certain functionalities or information.
                    It can exists in many shapes or forms, for instance:
```